content/en/Methodologies/ai-llm.md

```diff
@@ -6,17 +6,17 @@ description: >
   Review Cobalt pentest methodologies for an AI/LLM application.
 ---
 
-At Cobalt, we follow an industry-standard methodology based on the [OWASP Top 10 for Large Language Model Applications]([https://genai.owasp.org/llm-top-10/](https://owasp.org/www-project-top-10-for-large-language-model-applications/)). 
+Cobalt follows an industry-standard methodology based on the [OWASP Top 10 for Large Language Model Applications]([https://genai.owasp.org/llm-top-10/](https://owasp.org/www-project-top-10-for-large-language-model-applications/)). 
```

Comment:
The link leads to a 404, there are two URLs here for some reason.

Reply:
Thanks - fixed this. Going to just https://owasp.org/www-project-top-10-for-large-language-model-applications/ (which shouldn't 404) and removed the genai one.